source/interprocedural_analyses/taint/taintAnalysis.ml

```diff
@@ -348,19 +326,14 @@ let initialize_models
       ~definitions:(Some definitions_hashset)
       ~stubs:(Interprocedural.Target.HashsetSharedMemory.read_only stubs_shared_memory)
   in
-  Statistics.performance
-    ~name:"Parsed taint models"
-    ~phase_name:"Parsing taint models"
-    ~timer
-    ~integers:["models", Registry.size models; "queries", List.length queries]
-    ();
+  StepLogger.finish ~step_logger ~integers:[];
 
   let models, errors =
     match queries with
     | [] -> models, errors
     | _ ->
-        Log.info "Generating models from model queries...";
-        let timer = Timer.start () in
+        let step_logger = StepLogger.start ~start_message:"Generating models from model queries..." ~end_message:"Generatedmodels from model queries..." in
```

Comment:
same here. There is also a typo in the end message. It's somewhat important because we use the end message to query for metrics and alarms internally.
```suggestion
        let step_logger = StepLogger.start ~start_message:"Generating models from model queries" ~end_message:"Generated models from model queries" in
```